systems/overview/12-Glossary.md

```diff
@@ -3,91 +3,143 @@
 This section contains the most important domain and technical terms used when discussing the system.
 The goal is for all consumers of the architecture documentation to have an identical understanding of the terms and not use synonyms and homonyms.
 
-## Stakeholder
+## [Alias](https://www.wikidata.org/wiki/Help:Aliases)
 
-All persons, roles or organizations that:
+Alias (also _also known as_) is an alternative name for an [Entity](#entity). The usual or most important name is the [Label](#label). Aliases help people to find an item even if they don’t search with the label. For example, the item Q2 has the label “Earth” and aliases such as “Tellus” and “Blue Planet”. It's a type of [Term](#term).
```

Comment:
I only understood the part in the brackets after I opened the help page. Maybe we should copy the explanation in its full version from there.

Reply:
I actually copy-pasted the paragraph from here: https://www.wikidata.org/wiki/Wikidata:Glossary, but I do agree that the parts in bracket isn't very nice to the reader. :-)
Fixed.